frontend/.env.production

```diff
@@ -5,3 +5,5 @@ VITE_KEYCLOAK_REALMNAME=KAM_KEYCLOAK_REALMNAME
 VITE_KEYCLOAK_LOGIN_URL=KAM_KEYCLOAK_LOGIN_URL
 VITE_KEYCLOAK_LOGOUT_URL=KAM_KEYCLOAK_LOGOUT_URL
 VITE_KEYCLOAK_TOKEN_URL=KAM_KEYCLOAK_TOKEN_URL
+VITE_ROLEMAP_NAMESPACE=KAM_ROLEMAP_NAMESPACE
+VITE_ROLMAP_NAME=KAM_ROLMAP_NAME
```

Comment:
If I'm not wrong, VITE_ROLMAP_NAME can be normal. VITE_ROLEMAP_NAME

Reply:
You are wrong

Reply:
It will change even that? If not, why?

Reply:
A dobra, ja już straciłem resztki rozumu przez ten frontend. W każdym razie done